package.json

```diff
@@ -5,6 +5,11 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
+    "@fortawesome/fontawesome-svg-core": "^1.2.27",
+    "@fortawesome/free-brands-svg-icons": "^5.12.1",
+    "@fortawesome/free-regular-svg-icons": "^5.12.1",
+    "@fortawesome/free-solid-svg-icons": "^5.12.1",
+    "@fortawesome/react-fontawesome": "^0.1.8",
```

Comment:
Do you think we need this just for github button

Reply:
It can be used for many others things (e.g. #33 a back arrow). I think it could be interesting to keep it

Reply:
OK great